CONTRIBUTING.md

```diff
@@ -0,0 +1,162 @@
+# Contributing Guide
+
+* [Ways to contribute](#ways-to-contribute)
+* [Finding an issue](#finding-an-issue)
+* [Asking for help](#asking-for-help)
+* [Submitting a contribution](#submitting-a-contribution)
+* [Setting up a development environment](#setting-up-a-development-environment)
+* [Pull request checklist](#pull-request-checklist)
+
+Welcome to the DNF5 project! We are glad that you want to contribute!
+
+We welcome your feedback on different areas of our project, including the following:
+
+* Problems found during a new developer environment setup.
+* Gaps in the Quickstart Guide or other documentation.
```

Comment:
What is the `Quickstart Guide`? Is it referring to this docs or something else?

Reply:
it's inaccurate, right, it comes from the template I used. I would change it to "Gaps in our documentation"